dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild

```diff
@@ -126,7 +126,7 @@ src_compile() {
 }
```

Comment:
You shouldn't alter MAKEOPTS, just pass extra arguments to `emake`. I don't think the subst works with plain `-j` (no number). Use multiprocessing.eclass to parse MAKEOPTS.